build.sbt

```diff
@@ -51,14 +54,28 @@ lazy val `ReactiveMongo-Core` = project
     }
   )
 
-lazy val `ReactiveMongo` = new Driver(`ReactiveMongo-Core`).module
+
+lazy val `Actors-Akka` = project.in(file("actors-akka")).settings(
```

Comment:
SBT module naming

Reply:
What would you like the module name to be?

Reply:
all other modules are prefixed

Reply:
I assume you mean something like this? [Add ReactiveMongo as prefix to actor modules](https://github.com/ReactiveMongo/ReactiveMongo/pull/1245/commits/be05bbd8c6738d0631d96d5b2b166a78a88139d2)